apisix/plugins/wolf-rbac.lua

```diff
@@ -332,9 +333,9 @@ local function get_args()
     req_read_body()
     if string.find(ctx.var.http_content_type or "","application/json",
                    1, true) then
-        args, err = json.decode(ngx.req.get_body_data())
+        args, err = json.decode(req_get_body_data())
         if err then
-            core.log.error("json.decode(", ngx.req.get_body_data(), ") failed! ", err)
+            core.log.error("json.decode(", req_get_body_data(), ") failed! ", err)
```

Comment:
Would be better if we hold `req_get_body_data()` as a variable.

Reply:
done